components/PostForm/PostForm.js

```diff
@@ -0,0 +1,48 @@
+import styled from "styled-components";
+
+export function Postform() {
+  function handleClick(event) {
+    event.preventDefault();
+    document.getElementById("display").innerHTML =
+      document.getElementById("inputText").value;
```

Comment:
When using react, try to avoid manipulating the DOM directly. Rather try to use state (`useState`) and set it here.

```suggestion
    setValue(document.getElementById("inputText").value);
```

Reply:
Once you are using a form submit handler, you'll probably be better off using `event.target.elements.text.value` instead of `document.getElementById`.